src/adapter/src/catalog/transact.rs

```diff
@@ -2181,41 +2181,17 @@ impl Catalog {
                 }
                 Op::UpdateRotatedKeys {
                     id,
-                    previous_public_key_pair,
-                    new_public_key_pair,
+                    previous_public_key_pair: _,
+                    new_public_key_pair: _,
                 } => {
+                    // Noting to actually do but log.
```

Comment:
If this only logs, we should remove it. We could more easily log instead of call catalog_transact.

Reply:
Done.